### ISSUE TITLE
Suggestion for when copying links.

### ISSUE THREAD

**Issue by 54m4d** [rowrawer/stream-detector#193]
Hello

Is there any possibility to add a small animation that says `copied ✔` or something like that after a link is clicked?

Sometimes i find myself not sure if the link is copied, so what i do is i just click multiple times to make sure my link is copied.

<details>
<summary>Something like this</summary>
<img src=https://user-images.githubusercontent.com/52612348/236896394-6191d8ad-5f7b-4bdc-9e9e-c0c65c5bfc37.png>
</details>

**Comment by rowrawer** [comment]
This addon is not being actively developed anymore, but what you're talking about is already implemented in the form of notifications, unless you have them turned off in the options?

**Comment by 54m4d** [comment]
You are right, i turned them all off due to the annoyance they cause, specially on chromium browsers with the sound and all that.

If the development is stalled, then that's fine. 

Thank you for your support.

**Comment by 54m4d** [comment]
Right, i think that's an approach that i have tried too, but to be honest with you i don't want to have the windows notification panel full of 20 notifications saying "successfully copied the following links" it's quite aggressive IMO and i have to clear them out very often so that i can have room to see my other notifications, that's why i completely disabled notifications in the extension, other than that, i think a couple of extra clicks when copying a link won't hurt anyway since i'm used to it already.

But should this suggestion ever become current in the future, then at least we can find it described in this issue.
